docs/components/background.md

```diff
@@ -13,6 +15,9 @@ frustum culling issues when `a-sky` is further than the far plane of the
 camera. There are no unexpected occlusions either with far objects that might
 be behind of the sphere geometry of `a-sky`.
 
+Background supports the HTML color codes format, as well as the rgb(r,g,b), 
```

Comment:
Colors follow the THREE js format and apply to all properties of the color type. Maybe better to add a not here: https://aframe.io/docs/1.0.0/core/component.html#schema_color

Reply:
Will do. Sure